### PR TITLE
There was some code in the API client that didn't work properly

### PR DESCRIPTION
**Pull request by jakob223**
Some objects have data that has many properties - each property has a separate getter, which caches the results. It used to fetch new results from the data each time, appending it to the existing results, thus duplicating them once for each time the getter is called. I fixed all the occurrences of this.

Test by getting some piece of data and calling the same getter on it twice. For example,

```
tc = ThreatConnect(api_user,api_key,api_org_name,api_url)
data = tc.get_indicators_by_tag('strongarm','hosts').data()
print(data.hostName_list())
print(data.hostName_list()) # this should be the same as the previous output, not twice as long
```

### CODE REVIEW

File: ThreatConnect/ThreatConnect.py [hunk 1]
```diff
@@ -4128,14 +4180,18 @@ def description_list(self):
...
+        if self._dnsActive_list:
+            return self._dnsActive_list
         for data in self._data:
             if 'dnsActive' in data.keys():
                 self._dnsActive_list.append(data['dnsActive'])
 
         return self._dnsActive_list
 
-    #todo, this no longer works
+    #todo, this no longer works (why?, it's the exact same for all above and below!)
```
Comment (dicato) [inline ThreatConnect/ThreatConnect.py]: Comments such as these should not be in code we intend to release back to the maintainers. Given we did not write the code, we can only assume why the original comment exists. If it seems important, you can politely ask about it when issuing a pull request upstream.

**Comment by clokep** [comment]
@jake223 Please add a longer description about what this is actually fixing. What didn't work properly about it? What happened previously? What is the new behavior? How does one test this?

I'd also like to see the merge commit rebased out.

**Comment by jake223** [comment]
How do I rebase them out?

**Comment by jakob223** [comment]
closing old PRs

**Pull request closed without merging**